File: readability/readability.c
#include <cs50.h>
#include <ctype.h>
#include <math.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    char *text = get_string("Text: ");
    int Letters = 0;
    int Sentences = 0;
    int Words = 1; // +1 for the final word
    int length = strlen(text);
    char aux;
    for (int i = 0; i < length; i++)
    {
        if (isalpha(text[i]))
        {
            Letters++;
        }
        else if ((text[i] == '?') || (text[i] == '.') || (text[i] == '!'))
        {
            Sentences++;
        }
        else if (text[i] == ' ')
        {
            Words++;
        }
    }
    float L = ((float) Letters / (float) Words) * 100;
    float S = ((float) Sentences / (float) Words) * 100;
    float index = 0.0588 * L - 0.296 * S - 15.8;
    int grade = (int) round(index);

    if (grade > 16)
    {
        printf("Grade 16+\n");
    }
    else if (grade < 1)
    {
        printf("Before Grade 1\n");
    }
    else
    {
        printf("Grade %i\n", grade);
    }
}

File: sentimental-readability/readability.py
text = input("text: ")
letters = 0
words = 1
sentences = 0
for i in text:
    if (i.isalpha()):
        letters += 1
    elif (i == " "):
        words += 1
    elif (i == "." or i == "?" or i == "!"):
        sentences += 1
Le = (letters / words) * 100
Se = (sentences / words) * 100
index = 0.0588 * Le - 0.296 * Se - 15.8
index = round(index)
if index > 16:
    print("Grade 16+")
elif index < 1:
    print("Before Grade 1")
else:
    print(f"Grade {index}")


for i in range(3):
    i == 4
    break
else:
    